src/get-started/fwe/layout.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Layouts
+description: Learn how to create layouts in Flutter.
+prev:
+  title: Flutter fundamentals
+next:
+    title: State management
+---
+
+Given that Flutter is a UI toolkit, you’ll spend a lot of time creating layouts
+with Flutter widgets. In this section, you’ll learn how to build simple and
+complex layouts in Flutter with some of the most common layout widgets in
+Flutter. You’ll also learn about how to use DevTools to understand how your apps
+are being laid out, and you’ll learn how to debug one of the most common Flutter
+errors, which is related to the most common Flutter layout widgets.
+
+### Introduction to layouts
+
+The following tutorial will introduce you to layouts in Flutter, and give you an
+opportunity to work with the most common layout widgets, like rows and columns.
+
+<i class="material-symbols" aria-hidden="true">label</i> Tutorial: [Layouts in Flutter][]
+
+### Constraints
+
+Understanding “constraints” in Flutter are an important part of understanding
+how Flutter’s layout algorithm works.
+
+The following resources will teach you what constraints are, some widgets that
+allow you to work with constraints, and how to deal with “unbounded constraints”
```

Comment:
I'm wondering. Does these resources actually deal with unbounded restraints? Because you have that covered in the last bit of this page.